src/components/appNavigation/AppNavigation.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect } from "react";
+import "./AppNavigation.css";
+import { useState } from "react";
+import { useLocation } from "react-router-dom";
+import { IconButton, Link, Typography } from "@mui/material";
+import contacts from "../../assets/images/appNavigation/contacts.png";
+import contactsRed from "../../assets/images/appNavigation/contactsRed.png";
+import homeHealth from "../../assets/images/appNavigation/home_health.png";
+import homeHealthRed from "../../assets/images/appNavigation/home_healthred.png";
+import prescription from "../../assets/images/appNavigation/prescriptions.png";
+import prescriptionRed from "../../assets/images/appNavigation/prescriptionsRed.png";
+
+const navItems = [
```

Comment:
remove that declaration here and create file name may be `navigationData.ts` and define the interface of ItemNavigation and declare and export the dataItem array.